### PR TITLE
Fixed smoke test

### PR DESCRIPTION
**Pull request by kimlep01**
* Fixed smoke test trying to unstash not built combinations
* Use Raas names instead of ports + https
* Enabled smoke tests by default

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -218,8 +218,9 @@ def run_smoke(targets, toolchains, radioshields, meshinterfaces, raasPort, suite
                 def toolchain = toolchains.keySet().asList().get(j)
                 def radioshield = radioshields.get(k)
                 def meshInterface = meshinterfaces.get(l)
-                // Skip unwanted combination
-                if (target == "NUCLEO_F401RE" && toolchain == "IAR") {
+                // Skip unwanted combination & Wi-SUN compiled only with SL2P shield and vice versa
+                if ((target == "NUCLEO_F401RE" && toolchain == "IAR") || ("${meshInterface}" == "ws" && "${radioshield}" != "S2LP")
```
Comment (artokin) [inline Jenkinsfile]: Why Wi-SUN tests are skipped?

File: Jenkinsfile [hunk 2]
```diff
@@ -20,13 +20,13 @@ echo "Run smoke tests: ${params.smoke_test}"
 // Radios: Atmel & MCR20A
 // configurations: 6LP + Thread
 def raas = [
-  "lowpan_mesh_minimal_smoke_k64f_atmel.json": "8001",
+  "lowpan_mesh_minimal_smoke_k64f_atmel.json": "rauni",
 //  "lowpan_mesh_minimal_smoke_k64f_mcr20.json": "8034",
```
Comment (artokin) [inline Jenkinsfile]: Can commented lines be removed now?

File: Jenkinsfile (hunk 1 above)
Comment (kimlep01) [reply Jenkinsfile]: This follows the build step logic ( https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/master/Jenkinsfile#L83 ) where Wi-SUN interface is only built with S2LP and vice versa. This was broken previously as the test step tried to unstash all possible combinations.

Wi-SUN smoke tests are missing from this Jenkinsfile.

File: Jenkinsfile (hunk 2 above)
Comment (kimlep01) [reply Jenkinsfile]: Removed extra lines.

**Pull request merged**